azkaban-exec-server/src/main/java/azkaban/common/ServerUtils.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+
+package azkaban.common;
+
+import azkaban.Constants;
+import azkaban.execapp.event.JobCallbackManager;
+import azkaban.utils.Props;
+import org.apache.log4j.Logger;
```

Comment:
Should it be slf4j ?

Reply:
We are eventually going to move to log4j2 and with that slf4j is not useful. slf4j is anyway a facade and it relies on underlying implementation which is log4j for us.
For basic logging using slf4j would work, however, we make extensive use of appenders and while working on migration, it became really hard to work with slf4j. There were similar views I found on stackoverflow as well.